ivy/functional/backends/numpy/statistical.py

```diff
@@ -1,6 +1,14 @@
 # global
 import numpy as np
 from typing import Union, Optional, Sequence
+from ivy.func_wrapper import with_unsupported_dtypes
+from ivy.functional.frontends.numpy.func_wrapper import (
```

Comment:
please remove these imports if you are not using them

Reply:
Done